Gemfile

```diff
@@ -68,5 +68,6 @@ group :plugins do
   gem 'typogruby'
   gem 'uglifier'
   gem 'w3c_validators'
+  gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Comment:
Instead of the `if` I use this in my Gemfiles:
```rb
install_if(-> { Gem.win_platform? }) do
  gem 'wdm'
end
```

IIRC this leads to a consistent lockfile across OSes, whereas the dep is only installed when needed.

Reply:
I picked whatever `listen` recommended.

(I think it’s fine in this case, because the Gemfile.lock isn’t bundled with Nanoc anyway.)

Reply:
👍